### DISCUSSION TITLE
Unable to delete a project

### DISCUSSION THREAD

**Discussion by patelsaajan** [vercel/community#4313] [Help] [answered]
### Summary

I can not delete one of my projects. I go to setting/general then scroll to the bottom and click on delete. I am then asked to enter the project name twice, for example "johnsmith/project", and then click submit. However, everytime i click submit i am told that what i have written does not match the requested format yet ?! :/

### Steps to Reproduce

<img width="502" alt="Screenshot 2023-10-02 at 00 21 18" src="https://github.com/vercel/community/assets/85790308/8ff731cf-c048-4bd1-addf-210c41fb9081">

**Comment by thjiang** [comment, marked as answer]
Just type "delete my project" in the second input
